MonoGame.Framework.Content.Pipeline/Processors/TextureProcessorOutputFormat.cs

```diff
@@ -26,9 +28,28 @@ public enum TextureProcessorOutputFormat
 
         /// <summary>
         /// The SurfaceFormat value, of the input TextureContent object, is converted to an appropriate compressed format for the target platform.
-        /// This can include PVRTC for iOS, DXT for Windows 8 and Windows Phone, and ETC1 for Android.  Some Android devices can also support additional
-        /// compression formats, including PVRTC and DXT plus ATITC and other formats.
+        /// This can include PVRTC for iOS, DXT for desktop, Windows 8 and Windows Phone 8, and ETC1 or BGRA4444 for Android.
         /// </summary>
         Compressed,
+
+        /// <summary>
+        /// The pixel depth of the input texture is reduced to BGR565 for opaque textures, otherwise it uses BGRA4444.
+        /// </summary>
+        Color16Bit,
+
+        /// <summary>
+        /// The input texture is compressed using ETC1 texture compression.  Used on Android platforms.
+        /// </summary>
+        Etc1Compressed,
+
+        /// <summary>
+        /// The input texture is compressed using PVR texture compression. Used on iOS and some Android platforms.
+        /// </summary>
+        PvrCompressed,
+
+        /// <summary>
+        /// The input texture is compressed using ATI texture compression.  Used on some Android platforms.
+        /// </summary>
+        AtcCompressed,
```

Comment:
You could use a specific compression type, for example if your game was
targeting Tegra devices only you will want to use DXT rather than ETC1 and
BGRA4444.